lib/talis-cdk-stack/talis-cdk-stack.ts

```diff
@@ -11,6 +13,23 @@ export class TalisCdkStack extends Stack {
     if (props.vpcId) {
       this.vpc = this.resolveVpc(props.vpcId);
     }
+
+    cdk.Tags.of(scope).add("tfs-app", props.app); // e.g. depot
+    cdk.Tags.of(scope).add("tfs-environment", props.deploymentEnvironment); // e.g. production staging development
+    cdk.Tags.of(scope).add("tfs-release", props.release); // e.g. 8561-105814f
+
+    // props.env comes from aws cdk core StackProps. It's optional
+    // so that an environment agnostic stack can be created.
+    // If we create an environment agnostic stack - it can not contain
+    // region or service.
+    // However, when we deploy into our environments for real - props.env will be set
+    if (props.env?.region) {
+      cdk.Tags.of(scope).add("tfs-region", props.env.region);
```

Comment:
for `tfs-region` we meant the short region:
https://github.com/talis/talis-cdk-constructs/blob/902907814f31b61fb34fa7e24d845f87441a7829/lib/talis-cdk-stack/talis-region.ts#L7-L11

Reply:
agreed and updated